setup.py

```diff
@@ -44,7 +44,7 @@
 					]),
 	classifiers = classifiers,
 	packages=["zfstools"],
-	scripts=["bin/zreplicate", 'bin/zfs-shell', 'bin/zsnap', 'bin/zbackup'],
+	scripts=["bin/zreplicate", 'bin/zfs-shell', 'bin/zsnap', 'bin/zbackup', 'bin/zlock'],
```

Comment:
transfer-zfs-filesystem is not here.  Perhaps you'd like to remove it from the pull request?